inst/tests/tests.Rraw

```diff
@@ -18315,3 +18315,26 @@ if (test_bit64) {
   apple = data.table(id = c("a", "b", "b"), time = c(1L, 1L, 2L), y = i64v[1:3])
   test(2248, dcast(apple, id ~ time, value.var = "y"), data.table(id = c('a', 'b'), `1` = i64v[1:2], `2` = i64v[4:3], key='id'))
 }
+
+# trunc.char misbehaves for multibyte characters, #5096
+# Tests for combining characters
+options(datatable.prettyprint.char=4L)
+DT = data.table("áááá")
+test(2248.1, capture.output(print(DT)), c("     V1", "1: áááá"))
```

Comment:
```suggestion
test(2248.1, print(DT), output="1: áááá$")
```

And similarly for the other tests

Reply:
I needed to look this up, but `test()` always runs `print()` itself went `output=` is present, so we can omit the explicit `print()`:

https://github.com/Rdatatable/data.table/blob/40afa84c37063a17ec8a36e9eefa3507745d9923/R/test.data.table.R#L341

Moreover, I worry that `...` will get matched as a regex, which makes the test _possibly_ incorrect, so it's better to force matching `.` exactly with `[.]`